src/utils/logger.ts

```
@@ -0,0 +1,84 @@
+  error: 3,
+};
+
+class Logger {
+  private shouldLog(level: LogLevel): boolean {
+    return LEVELS[level] >= LEVELS[LOG_LEVEL as LogLevel];
+  }
+
+  private formatMessage(level: LogLevel, message: string, meta?: any): string {
+    const timestamp = new Date().toISOString();
+    const emoji = {
+      debug: '🔍',
```

ESLint

[error] Replace `'🔍'` with `"🔍"`

(prettier/prettier)